packages/react-components/react-rating-preview/stories/Rating/index.stories.tsx

```diff
@@ -6,7 +6,6 @@ import bestPracticesMd from './RatingBestPractices.md';
 export { Default } from './RatingDefault.stories';
 export { Color } from './RatingColor.stories';
 export { Max } from './RatingMax.stories';
-export { Mode } from './RatingMode.stories';
 export { Step } from './RatingStep.stories';
 export { Size } from './RatingSize.stories';
 export { Shape } from './RatingShape.stories';
```

Comment:
These don't need to be in alphabetical order. It's better to sort them from basic to advanced. For example:
* Controlled Value (recommend adding)
* Step
* Max
* Size
* Color
* Shape